website/api/web/WebsiteAPI.php

```diff
@@ -339,6 +339,10 @@ protected function botFile() {
         // Mark a new botfile for compilation if valid. Return error otherwise 
         if($this->isLoggedIn() && isset($_FILES['botFile']['name'])) {
             $user = $this->getLoggedInUser();
+
+            if(isset($this->config["compState"]["closeSubmissions"]) && $this->config["compState"]["closeSubmissions"]) {
```

Comment:
I tested with the config set to the values of true and false, which worked as expected.